lib/order.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "menu"
+
+class Order
+  attr_accessor :order, :total
+
+  def initialize 
+    @order = []
+    @total = 0
+  end
+
+  def add(dish)
+    @order << dish
+    @total += dish.values[0]
+    "Current total is #{@total}"
+  end
+
+  def show_order
+    @order.each { |dish| puts "#{dish.keys[0]} £#{dish.values[0]}" }
```

Comment:
I like how this works, it's nice and simple and clear. Do you need to use the index after .keys and .values if there's only one key and one value in the hash?